pkg/configs/advisory/v2/document.go

```diff
@@ -97,13 +97,38 @@ func (advs Advisories) Validate() error {
 		return fmt.Errorf("this file should not exist if there are no advisories recorded")
 	}
 
+	seenIDs := make(map[string]bool)
```

Comment:
I love that we can easily improve after the big chunk of CI you added in 23d, this is awesome